contrib/cirrus/build_podman.sh

```diff
@@ -2,13 +2,43 @@
 
 set -xeo pipefail
 
-mkdir -p "$GOPATH/src/github.com/containers/"
-cd "$GOPATH/src/github.com/containers/"
+#mkdir -p "$GOPATH/src/github.com/containers/"
+#cd "$GOPATH/src/github.com/containers/"
 
-systemctl stop podman.socket ||:
+systemctl stop podman.socket || :
 dnf erase podman -y
-git clone https://github.com/containers/podman.git
+#git clone https://github.com/containers/podman.git
+#
+#dnf install -y golang golang-bin gpgme-devel libseccomp-devel systemd-devel
+#
+#cd podman
+#make binaries
+#make docs
+#make install PREFIX=/usr
+#make install.systemd
 
-cd podman
-make binaries
-make install PREFIX=/usr
+PODMAN_TARBALL=podman-release-amd64.tar.gz
+TARGET_DIR=/var/tmp
+FILE_DIR=$TARGET_DIR/podman-v4.0.0-dev
+
+curl -SOsl \
+    --output-dir $TARGET_DIR \
+    --url "https://api.cirrus-ci.com/v1/artifact/github/containers/podman/Build%20for%20fedora-35/gosrc/$PODMAN_TARBALL"
+
+tar zxf $TARGET_DIR/$PODMAN_TARBALL
+
+SRC_DIR=$FILE_DIR/usr/bin
+install -m 755 $SRC_DIR/* /usr/bin
```

Comment:
`$FILE_DIR` is going to change anytime someone touches podman's `version.go`.  But there should only/ever be a single subdirectory.  So it's probably safer to install from `$TARGET_DIR/podman-v*/usr/bin/*` and let bash figure it out.  Similar for below.

Reply:
Another maybe easier thought...Lokesh is building a test RPM in podman CI, but we're not currently creating an artifact for it.  I could make that (simple) change, and get you the URL of the test `rpm` instead.

Reply:
https://github.com/containers/podman/pull/13344

Reply:
Lokesh has an even easier solution:  Apparently the latest from main is available in COPR, so just:

`dnf copr enable rhcontainerbot/podman-next -y`

and `dnf upgrade -y podman` :smile:

Reply:
@cevich I'll update the cirrus support scripts to pull from copr